content/module-2/5-testnet.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Topos Testnet
+description: The Topos testnet, overview and how-to
+---
+
+# Topos Testnet
+
+The Topos testnet is comprised of two subnets, Incal and Topos Subnet, along with a sequencer and an instance of the Executor Service. It provides all the infrastructure needed to deploy Smart Contracts to Incal and interact across subnets. The architecture of the testnet is identical to what you saw in the [Topos Playground Overview](./1-ERC20-Messaging.html).
+As mentioned [previously](./2-explorer.md), the Topos testnet explorer is available at [https://explorer.testnet-1.topos.technology](https://explorer.testnet-1.topos.technology).
+
+The Topos protocol is undergoing heavy development and features will be rolled out incrementally over time until mainnet is launched in 2024. The phases outlined below will change as detailed planning progresses.
+
+## Phase 1: Core Infrastructure
+
+This is where we are today. Phase 1 provides the basic building blocks and connects them all together, providing the foundation for further work and refinement. Not all components are fully functional (e.g. no ZK proofs, deterministic broadcasts rather than trhe more performant probabilistic flavour described in the whitepaper), but all of the main interfaces are there and operational, ready for developers and early adopters.
```

Comment:
```suggestion
This is where we are today. Phase 1 provides the basic building blocks and connects them all together, providing the foundation for further work and refinement. Not all components are fully functional (e.g. no ZK proofs, deterministic broadcasts rather than the more performant probabilistic flavour described in the whitepaper), but all of the main interfaces are there and operational, ready for developers and early adopters.
```